api/tests/test_api_spec.py

```diff
@@ -0,0 +1,156 @@
+from conftest import AUTH_COOKIE, DEFAULT_USERNAME, JOBS_URI, auth, login, make_job, submit_batch
+from flask_api import status
+
+
+def test_not_logged_in(client):
+    response = submit_batch(client)
+    assert response.status_code == status.HTTP_401_UNAUTHORIZED
+
+    response = client.get(JOBS_URI)
+    assert response.status_code == status.HTTP_401_UNAUTHORIZED
+
+    response = client.head(JOBS_URI)
+    assert response.status_code == status.HTTP_401_UNAUTHORIZED
+
+
+def test_logged_in_not_authorized(client):
```

Comment:
authorization only applies to `POST /jobs`, might move this test to `test_submit_jobs.py`, along with the granule name tests

Reply:
I was going for a model where the things that get to the handlers are in the respective files, things checked by the api are in `test_api_spec.py`. Auth is kinda weird because the logic is done before the handler, but sets a variable checked at the handler

Reply:
There's still some spec validation in `test_submit_jobs.py`, e.g. too many jobs, missing description, etc.  I feel more strongly about moving the granule name checks than moving the auth check, but moving both puts everything that needs to know about the structure of a new job payload into the one `test_submit_jobs.py` file.